### DISCUSSION TITLE
Explorer Submissions - Can't submit Wallet to explorer

### DISCUSSION THREAD

**Discussion by darrenkel** [WalletConnect/walletconnect-monorepo#3576] [Developer Support]
Hi,

I can't submit my wallet to the explorer because it does not implement the method `personal_sign` for the [sample DApp](https://lab.web3modal.com/with-wagmi/react)

How do I overcome this? I am able to connect accounts etc.

<img width="473" alt="image" src="https://github.com/WalletConnect/web3modal/assets/107671032/92fa414f-22ec-4e54-a3ad-9a8788ee5ffa">

**Comment by boidushya** [comment]
Hey @darrenvechain , is there any specific reason why your wallet doesn't support `personal_sign`? `personal_sign` is a pretty significant method for a wallet to respond to since `eth_sign` is deprecated.
